corehq/apps/domain/dbaccessors.py

```diff
@@ -34,7 +34,8 @@ def get_doc_ids_in_domain_by_type(domain, doc_type, database=None):
     )]
 
 
-def iterate_doc_ids_in_domain_by_type(domain, doc_type, chunk_size=10000, database=None):
+def iterate_doc_ids_in_domain_by_type(domain, doc_type, chunk_size=10000,
```

Comment:
can we just assume skip will be `1` if there exists a startkey_docid so we can have one less parameter?